tsl/test/sql/bgw_custom.sql

```diff
@@ -549,7 +549,7 @@ INSERT INTO sensor_data
 		random() AS cpu,
 		random()* 100 AS temperature
 	FROM
-		generate_series(:'start_date_sd'::timestamptz - INTERVAL '1 months', :'start_date_sd'::timestamptz - INTERVAL '1 week', INTERVAL '1 minute') AS g1(time),
```

Comment:
These inserts were taking a significant amount of time, so I reduced the amount of data being generated. I don't think this affects test behavior itself.